src/python/pants/backend/jvm/tasks/junit_run.py

```diff
@@ -294,11 +300,15 @@ def _spawn(self, distribution, executor=None, *args, **kwargs):
                                            executor=actual_executor,
                                            **kwargs)
 
+  @deprecated(removal_version='1.6.0.dev0',
+              hint_message='Subclasses should calculate the correct java distribution to use and '
```

Comment:
I re-read http://www.pantsbuild.org/deprecation_policy.html and you're right, I can nuke this. I was thinking we had settled on enclosing scope public implying nested scoped were public as well, but thankfully not!